spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/RewritePositionDeleteFilesSparkAction.java

```diff
@@ -298,7 +311,7 @@ private Result doExecuteWithPartialProgress(
         .run(fileGroup -> commitService.offer(rewriteDeleteFiles(ctx, fileGroup)));
     rewriteService.shutdown();
 
-    // Stop Commit service
+    // stop commit service
```

Comment:
Can we also slightly restructure `toGroupStream`? Spotless formatted it in a weird way. Should it also be private?

```
private Stream<RewritePositionDeletesGroup> toGroupStream(
      RewriteExecutionContext ctx,
      StructLikeMap<List<List<PositionDeletesScanTask>>> groupsByPartition) {

    return groupsByPartition.entrySet().stream()
        .filter(entry -> entry.getKey().size() > 0)
        .flatMap(
            entry -> {
              StructLike partition = entry.getKey();
              List<List<PositionDeletesScanTask>> fileGroups = entry.getValue();
              return fileGroups.stream().map(tasks -> newRewriteGroup(ctx, partition, tasks));
            })
        .sorted(RewritePositionDeletesGroup.comparator(rewriteJobOrder));
  }

  private RewritePositionDeletesGroup newRewriteGroup(
      RewriteExecutionContext ctx, StructLike partition, List<PositionDeletesScanTask> tasks) {

    int globalIndex = ctx.currentGlobalIndex();
    int partitionIndex = ctx.currentPartitionIndex(partition);
    FileGroupInfo info =
        ImmutableRewritePositionDeleteFiles.FileGroupInfo.builder()
            .globalIndex(globalIndex)
            .partitionIndex(partitionIndex)
            .partition(partition)
            .build();
    return new RewritePositionDeletesGroup(info, tasks);
  }

```

Reply:
Done